src/channel_state.js

```diff
@@ -94,28 +94,29 @@ export class ChannelState {
 		}
 	}
 
-	addReaction(reaction, reaction_counts, message) {
+	addReaction(reaction, message) {
 		const { messages } = this;
+		if (!message) return;
```

Comment:
Somehow it doesn't work without this line. Seems the function is called twice but only the second time it has `message`